src/shared/types/initRequestBody.ts

```diff
@@ -0,0 +1,17 @@
+import { hasStringProperty, nameOf } from "../utils/typeguards";
+
+export type InitRequestBody = Readonly<{
+  projectId: string;
+  language: string;
+  slugCodename: string;
+  algoliaAppId: string;
+  algoliaIndexName: string;
+}>;
+
+export const isValidInitRequestBody = (body: Record<string, unknown>): body is InitRequestBody =>
```

Comment:
Consider adding test for this method.

Reply:
Instead, I created a file (src/shared/constants/readmeSnippets.ts) where a created sample constants that are typed as the Configuration types. This way, whenever you change those types, TypeScript will tell you that you need to adjust it in this file and there is a comment in the file that reminds you to update the readme. In my opinion, this works (as a reminder) the same way a test would, but you will get the feedback faster than with tests.